pkg/database/realm.go

```diff
@@ -254,6 +254,7 @@ func (r *Realm) GetCurrentSigningKey(db *Database) (*SigningKey, error) {
 	var signingKey SigningKey
 	if err := db.db.
 		Model(r).
+		Where("realm_id = ?", r.ID).
```

Comment:
I still don't think this is right. `r` is the realm, not the signing key table. So these are conditions against the realms table. You either need to use related or drop to the signing key table instead:

```go
		Table("signing_keys").
		Where("realm_id = ?", r.ID).
		Where("active = ?", true).
		Order("signing_keys.created_at DESC").
```